### PR TITLE
[Mobile Config] Add deployment_info field to GatewayInfo

### PR DESCRIPTION
**Pull request by kurotych**

### CODE REVIEW

File: mobile_config/src/gateway_info.rs [hunk 1]
```diff
@@ -2,15 +2,99 @@ use chrono::{DateTime, TimeZone, Utc};
...
+    fn from(v: CbrsRadioDeploymentInfoProto) -> Self {
+        Self {
+            radio_id: v.radio_id,
+            elevation: v.elevation,
+        }
+    }
+}
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct RadioDeploymentInfo {
```
Comment (jeffgrunewald) [inline mobile_config/src/gateway_info.rs:66]: should probably also prepend this with `CbrsRadioDeploymentInfo` to avoid confusion

**Review by jeffgrunewald — APPROVED** [review]

File: mobile_config/src/gateway_info.rs [hunk 2]
```diff
@@ -2,15 +2,99 @@ use chrono::{DateTime, TimeZone, Utc};
...
+    /// CBSD_ID or radio
```
Comment (andymck) [inline mobile_config/src/gateway_info.rs:67]: what is the radio option here, is it radio_id, radio_serial ?

**Review by andymck — APPROVED** [review]

File: mobile_config/src/gateway_info.rs (hunk 1 above)
Comment (kurotych) [reply mobile_config/src/gateway_info.rs]: Fixed

File: mobile_config/src/gateway_info.rs (hunk 2 above)
Comment (kurotych) [reply mobile_config/src/gateway_info.rs]: I've removed confusing part taken from https://github.com/helium/helium-program-library/blob/22a2d00d556bc328f6126c865781d8361610522f/programs/helium-entity-manager/src/state.rs#L259

Comment (kurotych) [reply mobile_config/src/gateway_info.rs]: It can't be radio_serial (because of privacy). So I think it is cbsd_id only

**Pull request merged**